nipy/algorithms/statistics/models/tests/test_model.py

```diff
@@ -15,7 +16,7 @@
 
 N = 10
 X = np.c_[np.linspace(-1,1,N), np.ones((N,))]
-Y = np.r_[range(5), range(1,6)]
+Y = np.r_[list(range(5)), list(range(1,6))]
```

Comment:
From the docs: `np.r_` "Translates slice objects to concatenation along the first axis." Here's real slices:

    Y = np.r_[:5, 1:6]